packages/project-utils/__tests__/bundling/function/telemetryFunction.test.ts

```diff
@@ -1,5 +1,9 @@
-const path = require("path");
-const fs = require("fs");
+import * as fs from "fs";
```

Comment:
Right now, if a developer would run tests on his local machine, this test would break because they don't have WCP_APP_URL and WCP_API_URL environment variables set.

These env vars will be needed as long WCP is experimental, so let's make sure they're added.

Let's do the following. Let's edit the `example.env` env file, and ensure these env vars are present in it.

```
WCP_API_URL=...
WCP_APP_URL=...
```

This `example.env` file is always use to create the initial `.env` file when developers are creating a brand new project.

I'll make sure to add these to our CI/CD.